scripts/derivechangeset

```diff
@@ -0,0 +1,7 @@
+
+# HootApiDb requires a map layer, though (hence, 'blah').
+step1:
+	hoot derive-changeset $(HOOT_OPTS) -D convert.bounding.box=$(aoi) -D osm.changeset.sql.writer.generate.new.ids=false $(input1) $(input2) $(output) $(jobid) $(DB_URL)/blah
+
```

Comment:
Even with comment, not sure I understand what the last param is for.  Isn't the command deriving changeset in memory and writing to .osc file?